CRITICAL - 2025-08-04 15:20:34 --> ParseError: syntax error, unexpected token "{"
[Method: GET, Route: verEmpleados]
in APPPATH\Controllers\EmpleadoController.php on line 31.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp2\\htdocs\\repasoCodeigniter\\app\\Controllers\\EmpleadoController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\EmpleadoController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\EmpleadoController')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\EmpleadoController', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 15:26:24 --> ErrorException: preg_match(): Compilation failed: unmatched closing parenthesis at offset 26
[Method: GET, Route: eliminar_empleado/U003]
in SYSTEMPATH\Router\Router.php on line 436.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_match(): Compilation failed: unmatched closing parenthesis at offset 26', 'C:\\xampp2\\htdocs\\repasoCodeigniter\\system\\Router\\Router.php', 436)
 2 SYSTEMPATH\Router\Router.php(436): preg_match('#^eliminar_empleado\\([0-9]+)$#u', 'eliminar_empleado/U003', [])
 3 SYSTEMPATH\Router\Router.php(213): CodeIgniter\Router\Router->checkRoutes('eliminar_empleado/U003')
 4 SYSTEMPATH\CodeIgniter.php(832): CodeIgniter\Router\Router->handle('eliminar_empleado/U003')
 5 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 15:28:22 --> ErrorException: preg_match(): Compilation failed: unmatched closing parenthesis at offset 25
[Method: GET, Route: eliminar_empleado/U003]
in SYSTEMPATH\Router\Router.php on line 436.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_match(): Compilation failed: unmatched closing parenthesis at offset 25', 'C:\\xampp2\\htdocs\\repasoCodeigniter\\system\\Router\\Router.php', 436)
 2 SYSTEMPATH\Router\Router.php(436): preg_match('#^eliminar_empleado\\([^/]+)$#u', 'eliminar_empleado/U003', [])
 3 SYSTEMPATH\Router\Router.php(213): CodeIgniter\Router\Router->checkRoutes('eliminar_empleado/U003')
 4 SYSTEMPATH\CodeIgniter.php(832): CodeIgniter\Router\Router->handle('eliminar_empleado/U003')
 5 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 15:28:28 --> ErrorException: preg_match(): Compilation failed: unmatched closing parenthesis at offset 25
[Method: GET, Route: eliminar_empleado/U003]
in SYSTEMPATH\Router\Router.php on line 436.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_match(): Compilation failed: unmatched closing parenthesis at offset 25', 'C:\\xampp2\\htdocs\\repasoCodeigniter\\system\\Router\\Router.php', 436)
 2 SYSTEMPATH\Router\Router.php(436): preg_match('#^eliminar_empleado\\([^/]+)$#u', 'eliminar_empleado/U003', [])
 3 SYSTEMPATH\Router\Router.php(213): CodeIgniter\Router\Router->checkRoutes('eliminar_empleado/U003')
 4 SYSTEMPATH\CodeIgniter.php(832): CodeIgniter\Router\Router->handle('eliminar_empleado/U003')
 5 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 15:57:05 --> TypeError: App\Controllers\EmpleadoController::eliminar(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
[Method: GET, Route: eliminar_empleado/U010]
in APPPATH\Controllers\EmpleadoController.php on line 38.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->eliminar('U010')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 15:57:24 --> TypeError: App\Controllers\EmpleadoController::eliminar(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
[Method: GET, Route: eliminar_empleado/U010]
in APPPATH\Controllers\EmpleadoController.php on line 38.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->eliminar('U010')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 15:58:04 --> TypeError: App\Controllers\EmpleadoController::eliminar(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
[Method: GET, Route: eliminar_empleado/U011]
in APPPATH\Controllers\EmpleadoController.php on line 38.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->eliminar('U011')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 15:59:13 --> TypeError: App\Controllers\EmpleadoController::eliminar(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
[Method: GET, Route: eliminar_empleado/U012]
in APPPATH\Controllers\EmpleadoController.php on line 38.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->eliminar('U012')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 16:00:52 --> ErrorException: Undefined variable $datos
[Method: GET, Route: verEmpleados]
in APPPATH\Views\empleados.php on line 81.
 1 APPPATH\Views\empleados.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp2\\htdocs\\repasoCodeigniter\\app\\Views\\empleados.php', 81)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp2\\htdocs\\repasoCodeigniter\\app\\Views\\empleados.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('empleados', [], true)
 5 APPPATH\Controllers\EmpleadoController.php(14): view('empleados', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 16:00:56 --> ErrorException: Undefined variable $datos
[Method: GET, Route: verEmpleados]
in APPPATH\Views\empleados.php on line 81.
 1 APPPATH\Views\empleados.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp2\\htdocs\\repasoCodeigniter\\app\\Views\\empleados.php', 81)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp2\\htdocs\\repasoCodeigniter\\app\\Views\\empleados.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('empleados', [], true)
 5 APPPATH\Controllers\EmpleadoController.php(14): view('empleados', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 16:01:11 --> TypeError: App\Controllers\EmpleadoController::eliminar(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
[Method: GET, Route: eliminar_empleado/U013]
in APPPATH\Controllers\EmpleadoController.php on line 38.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->eliminar('U013')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-04 16:41:00 --> mysqli_sql_exception: Duplicate entry '54848778' for key 'telefono' in C:\xampp2\htdocs\repasoCodeigniter\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\repasoCodeigniter\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp2\htdocs\repasoCodeigniter\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\repasoCodeigniter\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\repasoCodeigniter\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp2\htdocs\repasoCodeigniter\app\Controllers\EmpleadoController.php(27): CodeIgniter\Model->insert(Array)
#8 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(933): App\Controllers\EmpleadoController->agregarEmpleado()
#9 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
#10 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\repasoCodeigniter\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\repasoCodeigniter\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\repasoCodeigniter\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-04 16:41:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '54848778' for key 'telefono'
[Method: POST, Route: agregarEmpleado]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (:codigo_usuario:, :nombre:, :apellido:, :telefono:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\EmpleadoController.php(27): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->agregarEmpleado()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 16:41:00 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '54848778' for key 'telefono'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (\'U009\', \'Rafael\', \'Javier\', \'54848778\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (\'U009\', \'Rafael\', \'Javier\', \'54848778\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (:codigo_usuario:, :nombre:, :apellido:, :telefono:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\EmpleadoController.php(27): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->agregarEmpleado()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 16:41:00 --> [Caused by] mysqli_sql_exception: Duplicate entry '54848778' for key 'telefono'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (\'U009\', \'Rafael\', \'Javier\', \'54848778\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (\'U009\', \'Rafael\', \'Javier\', \'54848778\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (\'U009\', \'Rafael\', \'Javier\', \'54848778\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`codigo_usuario`, `nombre`, `apellido`, `telefono`) VALUES (:codigo_usuario:, :nombre:, :apellido:, :telefono:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\EmpleadoController.php(27): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->agregarEmpleado()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-04 17:03:12 --> mysqli_sql_exception: Unknown column 'empleado_id' in 'where clause' in C:\xampp2\htdocs\repasoCodeigniter\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\repasoCodeigniter\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\repasoCodeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\repasoCodeigniter\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\repasoCodeigniter\app\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
#8 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
#9 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\repasoCodeigniter\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp2\htdocs\repasoCodeigniter\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp2\htdocs\repasoCodeigniter\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-04 17:03:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'empleado_id' in 'where clause'
[Method: GET, Route: update_empleado/U005]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = :empleado_id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 17:03:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'empleado_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = :empleado_id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 17:03:12 --> [Caused by] mysqli_sql_exception: Unknown column 'empleado_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = :empleado_id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-04 17:03:16 --> mysqli_sql_exception: Unknown column 'empleado_id' in 'where clause' in C:\xampp2\htdocs\repasoCodeigniter\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\repasoCodeigniter\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\repasoCodeigniter\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\repasoCodeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\repasoCodeigniter\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\repasoCodeigniter\app\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
#8 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
#9 C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\repasoCodeigniter\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp2\htdocs\repasoCodeigniter\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp2\htdocs\repasoCodeigniter\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-04 17:03:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'empleado_id' in 'where clause'
[Method: GET, Route: update_empleado/U005]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = :empleado_id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 17:03:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'empleado_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = :empleado_id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 17:03:16 --> [Caused by] mysqli_sql_exception: Unknown column 'empleado_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `empleado_id` = \'U005\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `empleado_id` = :empleado_id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\EmpleadoController.php(43): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->buscar('U005')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 17:29:01 --> ArgumentCountError: Too few arguments to function App\Controllers\EmpleadoController::editar(), 0 passed in C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: editar_empleado]
in APPPATH\Controllers\EmpleadoController.php on line 46.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->editar()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-04 17:30:31 --> ArgumentCountError: Too few arguments to function App\Controllers\EmpleadoController::editar(), 0 passed in C:\xampp2\htdocs\repasoCodeigniter\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: editar_empleado]
in APPPATH\Controllers\EmpleadoController.php on line 46.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmpleadoController->editar()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmpleadoController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
